zulip/zulip/__init__.py

```diff
@@ -1278,14 +1278,17 @@ def remove_alert_words(self, alert_words: List[str]) -> Dict[str, Any]:
             }
         )
 
-    def list_subscriptions(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
+    def list_subscriptions(self, request: Optional[Dict[str, Any]] = None, **kwargs: Any) -> Dict[str, Any]:
         '''
             See examples/list-subscriptions for example usage.
         '''
+        if request is not None:
+            logging.warning('')
```

Comment:
Does <code>include_subscribers as a dictionary parameter will be deprecated in future versions .</code> looks like a fine warning ? Am open to other suggestions as well .